src/read.c

```diff
@@ -653,9 +653,24 @@ typedef struct avifTile
     uint32_t width;  // Either avifTrack.width or avifDecoderItem.width
     uint32_t height; // Either avifTrack.height or avifDecoderItem.height
     uint8_t operatingPoint;
+    uint8_t type[4]; // "av01" for AV1 ("av02" for AV2 if AVIF_CODEC_AVM)
```

Comment:
this should be a boolean (or an enum with AV1 and AV2 as possible values for future-proofing).

so that the type of the tile can be checked with a simple `==` instead of using a `memcmp` every time.

Reply:
Done in `read.c` and `write.c` with `avifCodecType` enum.